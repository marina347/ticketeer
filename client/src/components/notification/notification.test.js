import React from "react";
import { shallow } from "enzyme";
import Notification from "./notification.component";

describe("Notification component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Notification />);
  });

  it("Should render Notification component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
