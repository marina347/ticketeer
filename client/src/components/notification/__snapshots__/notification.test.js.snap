// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification component Should render Notification component 1`] = `
<ToastContainer
  autoClose={5000}
  bodyClassName="toast-body"
  closeButton={[Function]}
  closeOnClick={true}
  draggable={true}
  draggablePercent={80}
  hideProgressBar={true}
  newestOnTop={false}
  pauseOnFocusLoss={true}
  pauseOnHover={true}
  position="bottom-right"
  role="alert"
  rtl={false}
  style={
    Object {
      "fontSize": "1.6rem",
    }
  }
  transition={[Function]}
/>
`;
